pkgs/args/pubspec.yaml

```diff
@@ -13,4 +13,4 @@ environment:
 
 dev_dependencies:
   dart_flutter_team_lints: ^3.0.0
-  test: ^1.16.0
+  test: ^1.16.6
```

Comment:
Just curious why the increase in specificity?

Reply:
It's what `dart pub upgrade --tighten` does 🤷